apps/AppMakefileMbed.mk.posix

```diff
@@ -0,0 +1,93 @@
+################################################################################
+## Linux flavored Makefile for getting Mbed code onto microcontrollers
+################################################################################
+
+# Set toolchain path if arm-none-eabi- binaries are not in user path
+# TOOLCHAIN_PATH ?=
+TERMINAL ?= gnome-terminal -e
+
+ifdef SEGGER_SERIAL
+JLINKEXE_OPTION = -SelectEmuBySn $(SEGGER_SERIAL)
+JLINKGDBSERVER_OPTION = -select USB=$(SEGGER_SERIAL)
+JLINKRTTCLIENT_OPTION = -select USB=$(SEGGER_SERIAL)
+endif
+
+MAKE_BUILD_FOLDER = mkdir -p $(OUTPUT_PATH)
+
+JLINK = -JLinkExe $(JLINK_OPTIONS) $(JLINKEXE_OPTION)
+JLINKGDBSERVER = JLinkGDBServer $(JLINK_OPTIONS) $(JLINKGDBSERVER_OPTION)
+JLINKRTTCLIENT = JLinkRTTClient $(JLINK_OPTIONS) $(JLINKRTTCLIENT_OPTION)
+
+clean:
+	rm -rf $(OUTPUT_PATH)
+	rm -f *.jlink
+	rm -f JLink.log
+	rm -f .gdbinit
+
+flash: all flash.jlink 
+	$(JLINK) $(OUTPUT_PATH)flash.jlink
+
+flash.jlink:
+	$(MAKE_BUILD_FOLDER)
+	printf "r\n" > $(OUTPUT_PATH)flash.jlink
+	printf "loadbin $(OUTPUT_PATH)/$(OUTPUT_BIN), $(FLASH_START_ADDRESS) \nr\ng\nexit\n" >> $(OUTPUT_PATH)flash.jlink
+
+recover: recover.jlink erase-all.jlink pin-reset.jlink
+	$(MAKE_BUILD_FOLDER)
+	$(JLINK) $(OUTPUT_PATH)recover.jlink
+	$(JLINK) $(OUTPUT_PATH)erase-all.jlink
+	$(JLINK) $(OUTPUT_PATH)pin-reset.jlink
+
+#recover.jlink:
+#	$(MAKE_BUILD_FOLDER)
+#	printf "si 0\nt0\nsleep 1\ntck1\nsleep 1\nt1\nsleep 2\nt0\nsleep 2\nt1\nsleep 2\nt0\nsleep 2\nt1\nsleep 2\nt0\nsleep 2\nt1\nsleep 2\nt0\nsleep 2\nt1\nsleep 2\nt0\nsleep 2\nt1\nsleep 2\nt0\nsleep 2\nt1\nsleep 2\ntck0\nsleep 100\nsi 1\nr\nexit\n" > $(OUTPUT_PATH)recover.jlink
+#
+#pin-reset.jlink:
+#	$(MAKE_BUILD_FOLDER)
+#	printf "w4 40000544 1\nr\nexit\n" > $(OUTPUT_PATH)pin-reset.jlink
+#
+#pin-reset: pin-reset.jlink
+#	$(MAKE_BUILD_FOLDER)
+#	$(JLINK) $(OUTPUT_PATH)pin-reset.jlink
+#
```

Comment:
What's going on with these?

Reply:
I didn't implement them because I didn't know how on the new chip...we could either remove them from the file entirely or work more to make them happen.

Reply:
Importantly though, the `recover` rule on line 35 depends on them.

Reply:
This is also really an issue with #32

Reply:
Oh, you're saying these changes will disappear from this PR after #32 is merged? That makes sense.

Reply:
I commented out recover